lib/common/util.js

```diff
@@ -360,6 +362,14 @@ function makeAuthenticatedRequestFactory(config) {
     if (stream) {
       return stream;
     }
+
+    return {
+      abort: function() {
+        if (activeRequest_) {
+          activeRequest_.abort();
```

Comment:
Can we document that? If it's async, I could see people running into a scenario where they try and abort but nothing happens.